stylesheets/commons/Fountain.less

```diff
@@ -460,6 +460,14 @@ The smaller sizes are for sub-sections. */
 	text-shadow: 0.0625rem 0.0625rem 0.125rem #000000;
 }
 
+.facetDesc {
+	border-top: 0.156rem solid #505050;
+
+	&:hover {
+		border-top: 0.156rem solid #999999;
```

Comment:
could possibly use the `--clr-moon-X0` css vars here
(am on phone so hard to check if we have fitting ones)

Reply:
Thank you for the var color list.
I'll pick one from that now, and will eventually replace most of the colors within the stylesheet as needed.